keps/sig-node/4010-head-request-probes/kep.yaml

```diff
@@ -0,0 +1,36 @@
+title: HEAD Request Probes
+kep-number: 4010
+authors:
+  - "@tobiasgiese"
+owning-sig: sig-node
+participating-sigs:
+  - sig-node
+status: provisional
+creation-date: 2023-05-16
+reviewers:
+  - TBD
+approvers:
+  - TBD
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.28"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.28"
+  beta: "v1.29"
+  stable: "v1.30"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: HeadRequestProbes
+    components:
+      - kube-apiserver
+      - kube-controller-manager
```

Comment:
I think the kubelet might need to implement this change too.

Reply:
100% - controller manager should not factor in but kubelet is where probes happen.